locations/spiders/pfg_fr.py

```
@@ -0,0 +1,29 @@
+from scrapy.http import Response
+
+from locations.camoufox_spider import CamoufoxSpider
+from locations.categories import Categories, apply_category
+from locations.items import Feature
+from locations.settings import DEFAULT_CAMOUFOX_SETTINGS
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class PfgFRSpider(StructuredDataSpider, CamoufoxSpider):
+    name = "pfg_fr"
+    item_attributes = {"brand": "PFG", "brand_wikidata": "Q3396087"}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)